### PR TITLE
whoops

### PR DESCRIPTION
**Pull request by ir0nlake**

### CODE REVIEW

File: src/main.c [hunk 1]
```diff
@@ -1,48 +1,53 @@
...
-} else {
-	fclose(fpt);
-	return 0;
+
 }
 
+int fileexist(char* filename) {
+FILE *fpt = fopen(filename,"rb");
+	if(!fpt){
+		return 1; // file doesn't exist
```
Comment (strdr1ft) [inline src/main.c]: should return 0

File: src/main.c [hunk 2]
```diff
@@ -1,48 +1,53 @@
...
+	} else {
+		fclose(fpt);
+		return 0;
```
Comment (strdr1ft) [inline src/main.c]: should return 1

File: src/main.c [hunk 3]
```diff
@@ -1,48 +1,53 @@
...
 
 int main() {
 	consoleDemoInit(); // setup the display for text
-	printf("NTRLDR (git %s, builddate %s %s)\nwritten by dr1ft/UTP\nhttps://discord.gg/CJZADTM\n\n",gitrev,__DATE__,__TIME__); // display copyright and build information
-	if(fatInitDefault()){ // init fat
+	printf("NTRLDR (git %s)\nwritten by dr1ft/UTP\nhttps://discord.gg/CJZADTM\n\n",gitrev); // display copyright and build information
+	if(fatInitDefault()) {
 
-        if(!fileexist("warned")){ // if the file doesnt exist, show our scary warning message
+        if(!fileexist("ubeenwarned")) {
```
Comment (strdr1ft) [inline src/main.c]: just use the filename that was already in the original version .-.

File: src/main.c [hunk 4]
```diff
@@ -56,12 +61,14 @@ int main() {
 	slot1[0x73] = (crc1 & 0xFF00) >> 8;
 	writeFirmware(0x3FE00,slot0,0x100); // write our modifications back to nvram
 	writeFirmware(0x3FF00,slot1,0x100);
-	puts("ok\nloading firmware.nds...");
+	puts("all ready, loading firmware.nds...");
```
Comment (strdr1ft) [inline src/main.c]: why edit this text?

File: src/main.c [hunk 5]
```diff
@@ -56,12 +61,14 @@ int main() {
...
 	int error = runNdsFile("/firmware.nds",0,0); // bootstrap firmware.nds
 	printf("\x1b[31;1merror %d\nmake sure firmware.nds is in\nthe root of the filesystem!", error); // if something fucks up display an error code
+
+	}
+	else {
+		printf("\x1b[31;1mFAT failed to init");
```
Comment (strdr1ft) [inline src/main.c]: again, why edit this

File: src/main.c (hunk 1 above)
Comment (ir0nlake) [reply src/main.c]: i return 0 since success in c is *usually* marked with 0, however I will change as requested.

File: src/main.c [hunk 6]
```diff
@@ -19,33 +19,31 @@ static unsigned char slot1[0x100];
...
-
 }
 
 int main() {
 	consoleDemoInit(); // setup the display for text
 	printf("NTRLDR (git %s)\nwritten by dr1ft/UTP\nhttps://discord.gg/CJZADTM\n\n",gitrev); // display copyright and build information
 	if(fatInitDefault()) {
 
-        if(!fileexist("ubeenwarned")) {
+        if(fileexist("/warning_shown")) {
```
Comment (strdr1ft) [inline src/main.c]: why did you remove the inversion???? this still fails for the same reason as the last commit

**Pull request merged**